Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json

```diff
@@ -116,9 +142,15 @@
                     "updated": 1638339266880
                 },
                 {
+                    "language": "TR",
+                    "text": "Aynı ticaret motoru, bir veri yapısının yalnızca düğüm tabanlı bir temsili olduğundan, birden fazla ticaret oturumunda kullanılabilir.",
+                    "updated": 1642504933300
+                   },
+                   {
```

Comment:
Those brackets need to be more left, maybe you can orient on already merged translations or files you also commited a bit more down. This occurs at some parts in this file.

Reply:
I did not fully understand. can you be a little more clear?